src/main/java/seedu/address/commons/exceptions/IllegalValueException.java

```diff
@@ -13,7 +13,7 @@ public IllegalValueException(String message) {
 
     /**
      * @param message should contain relevant information on the failed constraint(s)
-     * @param cause of the main exception
+     * @param cause   of the main exception
```

Comment:
extra space

Reply:
Part of the auto-formatter to make descriptions flushed. So "of" is aligned with "should"